Dockerfile.frontend

```diff
@@ -0,0 +1,13 @@
+FROM node:lts as builder
+COPY package*.json /data/
+WORKDIR /data
+RUN npm install
+ENV PATH /data/node_modules/.bin:$PATH
+
+COPY . /data/
+RUN npm run build
+
+FROM nginx:stable-alpine
+WORKDIR /opt/product-opener
+COPY --from=builder /data/node_modules/ /opt/product-opener/node_modules/
```

Comment:
Why do you want to copy the `node_modules` dir into the image? It's not actually required while running the frontend container, and the directory isn't exposed externally.

Reply:
Okay, I managed to use the frontend volumes for the backend to use in readonly ... But eventually we'll need to keep an eye on those `dist/` files, ideally the server should not need them at all, and caching should happen in Nginx not in the backend.